data/initrd/scripts/url/hmc/mount

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
\# Implement the 'hmc' URL scheme for linuxrc
\# See https://github.com/openSUSE/linuxrc/blob/a72ccf44eb19b6d55bc54058876fa5f64021eee6/user_defined_url_schemes.txt